controllers/dexserver_controller.go

```diff
@@ -107,52 +108,145 @@ func (r *DexServerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	log.V(1).Info("DexServer ", "name:", dexServer.Name, "namespace:", dexServer.Namespace)
```

Comment:
No concerns really if you want to leave this in, but just want to be sure you noticed you can see this info in all the other log statements already such as:
```
2021-10-07T11:27:35.815-0400	DEBUG	controller-runtime.manager.controller.dexserver	Reconciling...	{"reconciler group": "auth.identitatem.io", "reconciler kind": "DexServer", "name": "dex2", "namespace": "dex-operator"}
```

Reply:
removed that line